test/rocket-test.js

```diff
@@ -38,20 +38,24 @@ describe('Initial setup of rocket', function() {
 
 describe('Functionality of rocket', function() {
   it('Should be able to load astronauts', function() {
-    const rocket = new Rocket();
+    let rocket = createRocket();
 
-    rocket.loadAstronaut('Alan Shepard');
+    rocket = loadAstronaut(rocket, 'Alan Shepard');
 
     expect(rocket.astronauts).to.deep.equal(['Alan Shepard']);
   });
 
   it('Should be able to ignite the engines', function() {
-    const rocket = new Rocket();
+    let rocket = createRocket();
 
-    rocket.lightThisCandle();
+    rocket = lightThisCandle();
```

Comment:
I appreciate that you remembered the conversation around this after M1 midmods and implemented this test with that suggested style in mind! I had already forgotten about that until seeing this pattern here. 😂